core/workspace.js

```diff
@@ -328,7 +328,14 @@ Blockly.Workspace.prototype.getAllBlocks = function(ordered) {
       blocks.push.apply(blocks, blocks[i].getChildren(false));
     }
   }
-  return blocks;
+
+  // Insertion markers exist on the workspace for rendering reasons, but aren't
+  // "real" blocks from a developer perspective.
+  var filtered = blocks.filter(function(block) {
```

Comment:
Should these be filtered at the `getDescendent()` and `getChildren()` level? I imagine we don't want insertion markers returned via those methods, either.

Reply:
Hm, maybe?  I can't tell where the right place for this code is.